lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -53,6 +53,9 @@
 <!-- FAIL(errors-in-console): exception thrown -->
 <script type="text/javascript">throw new Error('An error');</script>
 
+<!-- Stack Pack Wordpress detection script -->
+<script src="fake-script-wp-includes"></script>
```

Comment:
do we need `type="just-for-library-detector"` in here?

Reply:
I suggested `type="example"` since it's specced to always be an example MIME type, but I guess it's unlikely there will ever be a `just-for-library-detector` type we'll collide with :)

Reply:
Took brendan's type="example" for this :smile: